pepper-apis/studybuilder-cli/tenants/register-user-in-pepper-action.js

```diff
@@ -169,9 +170,23 @@ exports.onExecutePostLogin = async (event, api) => {
         message: 'User need to register first in order to login',
         statusCode: 403,
       };
-      return api.access.deny(loginErrPayload.message);
+      return api.access.deny(JSON.stringify(loginErrPayload));
+    }
+
+    //added below in action to restrict access to other cmi studies in same tenant
+    if (pepper_params.mode && pepper_params.mode === "login" && event.user.app_metadata.user_guid
```

Comment:
check will avoid call to backend and will restrict from user trying to login if not registered to the study